lib/luatest_server.py

```diff
@@ -71,6 +73,7 @@ def __init__(self, _ini=None, test_suite=None):
         self.testdir = os.path.abspath(os.curdir)
         self.vardir = ini['vardir']
         self.builddir = ini['builddir']
+        self.lua_libs = ini['lua_libs']
```

Comment:
Am I understand right that we'll miss ability to run the test using `luatest` directly?

Reply:
From tarantool test directory we always run luatests using test-run.py (as i know). I add this option for this case. If someone want to run tests which use custom lua modules using luatest directly he should install this modules in the directory where `require` can find it.

Reply:
> From tarantool test directory we always run luatests using test-run.py (as i know).

I often run `luatest -v -p foo` directly to run a particular use case. Another reason to keep tests luatest compatible is motivation behind implementing luatest support in test-run: sooner or later we want to fully shift to luatest.

> If someone want to run tests which use custom lua modules using luatest directly he should install this modules in the directory where `require` can find it.

Just place the Lua file somewhere in `test/` and require as `require('test.<...>')`. It works both with luatest under test-run and pure luatest.